esiclient/tests/unit/v1/test_switch.py

```diff
@@ -263,6 +263,62 @@ def mock_gfnifp_call(np, n_dict, client):
             mock_gfnifp.call_count, 2)
 
 
+class TestListSwitches(base.TestCommand):
+
+    def setUp(self):
+        super(TestListSwitches, self).setUp()
+        self.cmd = switch.ListSwitches(self.app, None)
+
+        self.port1 = utils.create_mock_object({
+            "uuid": "port_uuid_1",
+            "node_uuid": "11111111-2222-3333-4444-aaaaaaaaaaaa",
+            "local_link_connection": {'switch_info': 'switch1',
+                                      'switch_id': 'e4:c7:22:c0:0b:69'},
+            "internal_info": {'tenant_vif_port_id': 'neutron_port_uuid_1'},
+        })
+        self.port2 = utils.create_mock_object({
+            "uuid": "port_uuid_2",
+            "node_uuid": "11111111-2222-3333-4444-bbbbbbbbbbbb",
+            "local_link_connection": {'switch_info': 'switch1',
+                                      'switch_id': 'e4:c7:22:c0:0b:69'},
+            "internal_info": {},
+        })
```

Comment:
Super good that you are testing duplicate switches!

Reply:
Thanks!